selfdrive/sensord/sensors/bmx055_gyro.cc

```diff
@@ -26,6 +27,23 @@ int BMX055_Gyro::init() {
     goto fail;
   }
 
+  // assumed to be exported on boot
+  if (gpio_init(gpio_nr, false) != 0) {
+    ret = -1;
+    goto fail;
+  }
+
+  if (gpio_set_edge(gpio_nr, Edgetypes::Rising) != 0) {
+    ret = -1;
+    goto fail;
+  }
+
+  gpio_fd = gpio_get_ro_value_fd(gpio_nr);
+  if (gpio_fd < 0) {
+    ret = -1;
+    goto fail;
+  }
```

Comment:
this block is repeated a few times, can we pull this into a helper function or something?